development/sentry-upload-artifacts.sh

```diff
@@ -0,0 +1,97 @@
+#!/usr/bin/env bash
+
+set -x
+set -e
+set -u
+set -o pipefail
+
+readonly __SCRIPT_NAME__="${0##*/}"
+readonly __SEE_HELP_MESSAGE__="See '${__SCRIPT_NAME__} --help' for more information."
+
+function die {
+  local message="${1}"
+
+  printf "ERROR: %s\\n" "${message}" >&2
+
+  exit 1
+}
+
+function show_help {
+  cat << EOF
+${__SCRIPT_NAME__}"
+Upload JavaScript bundles and sourcemaps to Sentry
+
+Options:
+  -h, --help               Show help text
+  -r, --release <release>  Sentry release to upload files to (defaults to 'VERSION' environment variable)
+EOF
+}
+
+function upload_bundles {
+  local release="${1}"; shift
+
+  for filepath in ./dist/chrome/*.js
+  do
+    if [[ -f $filepath ]]
+    then
+      upload_bundle "${release}" "${filepath}"
+    fi
+  done
+}
+
+function upload_bundle {
+  local release="${1}"; shift
+  local filepath="${1}"; shift
+  local filename
+
+  filename="$( basename "${filepath}" )"
+
+  printf 'Uploading %s' "${filename}"
```

Comment:
Nit: do we need a newline here?

```suggestion
  printf 'Uploading %s\n' "${filename}"
```

Reply:
Ah, yes I think we do! Good catch.